src/components/layout/nav.js

```diff
@@ -796,10 +796,12 @@ export const NavInterim = ({ interim_type }) => (
                     </MobileWrapper>
                 </Flex>
                 <Auto jc="flex-end" ai="center">
-                    <LanguageSwitcher short_name="true" />
-                    <LeftButton secondary to="/">
-                        {localize('Explore Deriv.com')}
-                    </LeftButton>
+                    {!landing_type && <LanguageSwitcher short_name="true" />}
```

Comment:
The `landing_type` check an be on its parent component `Auto` directly instead of the 2 childs.

Reply:
done